llvm/lib/Target/AMDGPU/SIInstrInfo.td

```diff
@@ -1987,20 +1989,29 @@ class getIns64 <RegisterOperand Src0RC, RegisterOperand Src1RC,
                       (ins Src2Mod:$src2_modifiers, Src2RC:$src2),
                       (ins Src2RC:$src2)),
                  (ins));
-  dag clamp = !if(HasClamp, (ins Clamp0:$clamp), (ins));
+  // If there is vdst_in after clamp with HasFP8DstByteSel we cannot use
```

Comment:
I can change the placement of the byte_sel operands, but that is a big disruption. If needed it will require a separate change.